### DISCUSSION TITLE
Additional point system

### DISCUSSION THREAD

**Discussion by ykdojo** [csdojo-defaang/dojo-bot#11] [General]
I've noticed that sometimes people help each other in regular channels, for example, #learn-to-code.

So my suggestion would be to give a fraction of a point for every message someone sends.

Maybe 0.1 points per message no matter where it is, whether it's on a forum or a regular channel... What do you think?

**Comment by mezotv** [comment]
Sounds good but i think a cooldown would be good as well to prevent spam.

**Reply by ykdojo** [reply]
I'm not sure if that's necessary actually. If someone spams, we can just kick/ban them?

**Reply by mezotv** [reply]
True that yea.

**Reply by cybardev** [reply]
> I'm not sure if that's necessary actually. If someone spams, we can just kick/ban them?

@ykdojo I think it is a good idea. Because some people just send a few words at a time.
```
Like
this, maybe
if you know
what I mean.
```
They shouldn't be punished for valid engagement, but rather instead of gaining 0.1 points for each line, they can get 0.1 points for each bundle of messages within a given timespan, like 90 seconds.

**Reply by mezotv** [reply]
> They shouldn't be punished for valid engagement, but rather instead of gaining 0.1 points for each line, they can get 0.1 points for each bundle of messages within a given timespan, like 90 seconds.

Yea that's what I meant as well. Thanks, cybardev!
